src/hooks/useNote.js

```diff
@@ -0,0 +1,52 @@
+import { useState, useEffect } from 'react'
+
+function useNote({ serviceClient, noteId }) {
+  // `docId` is the id of the note for which we have a state.
+  // `noteId` is the id of the requested note.
+  const [docId, setDocId] = useState(noteId)
+  const [loading, setLoading] = useState(true)
+  const [doc, setDoc] = useState(undefined)
+  const [title, setTitle] = useState(undefined)
+
+  // reload if ever noteId changes
+  useEffect(
+    () => {
+      if (docId !== noteId) {
+        setLoading(true)
+        setTitle(undefined)
+        setDoc(undefined)
+        setDocId(noteId)
+      }
```

Comment:
Does this hook make sense without the other `useEffect` in this file? Would it make sense to combine them into a single `useEffect` call?